src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -1579,6 +1579,11 @@ static inline int freeze_internal(JavaThread* current, intptr_t* const sp) {
     verify_continuation(cont.continuation());
     freeze_result res = entry->is_pinned() ? freeze_pinned_cs : freeze_pinned_monitor;
     log_develop_trace(continuations)("=== end of freeze (fail %d)", res);
+    // Avoid Thread.yield() loops without safepoint polls (see 8335269).
```

Comment:
Is an explicit reference to a JBS issue id like this still recommended practice? After all, it will be a prefix in the merged commit message.

Reply:
Right, I removed it from the comment.